UM/Trust.py

```diff
@@ -435,6 +448,20 @@ def signedFolderCheck(self, path: str) -> bool:
                         if os.path.islink(dir_full_path) and not self._follow_symlinks:
                             Logger.log("w", "Directory symbolic link '{0}' will not be followed.".format(dir_full_path))
 
+                # Check if the files moved to storage are still correct.
+                if storage_json:
+                    for entry in storage_json:
+                        try:
+                            # If this doesn't raise an exception, it's correct.
+                            CentralFileStorage.retrieve(entry[1], entry[3], Version(entry[2]))
+                        except:
+                            self._violation_handler("Centrally stored file '{0}' didn't match with checksum or it could not be found".format(entry[1]))
```

Comment:
Maybe use the string formatting again?

```suggestion
                            self._violation_handler(f"Centrally stored file '{entry[1]}' didn't match with checksum or it could not be found")
```